extensions/typescript-language-features/src/utils/configuration.browser.ts

```diff
@@ -8,12 +8,20 @@ import { BaseServiceConfigurationProvider } from './configuration';
 
 export class BrowserServiceConfigurationProvider extends BaseServiceConfigurationProvider {
 
-	// On browsers, we only support using the built-in TS version
-	protected extractGlobalTsdk(_configuration: vscode.WorkspaceConfiguration): string | null {
+	protected extractGlobalTsdk(configuration: vscode.WorkspaceConfiguration): string | null {
```

Comment:
Can the implementation of these functions be moved into `BaseServiceConfigurationProvider` now?

Reply:
Done.